### DISCUSSION TITLE
Default settings.

### DISCUSSION THREAD

**Discussion by WilsonBradley** [rocksdanister/lively#645] [Q&A] [answered]
I did not see a Set Default option. Would like to go back to default settings under Performance section.
Could someone post a screenshot of the default settings?

I messed with all the settings while trying to resolve a known bug.

Thanks..

![image](https://user-images.githubusercontent.com/8173893/127200077-7ac097e1-8c79-4061-8b3b-1b4c11ce37f3.png)

**Comment by rocksdanister** [comment, marked as answer]
Application Fullscreen -> Pause, rest same as screenshot.

**Reply by WilsonBradley** [reply]
Thanks

**Comment by WilsonBradley** [comment]
Update:  Default settings below.
![image](https://user-images.githubusercontent.com/8173893/127207679-284394af-97b4-4ccd-ad99-acddfa984096.png)
